src/aura/Events/Events.evt

```diff
@@ -0,0 +1,3 @@
+<aura:event type="COMPONENT">
+    	<aura:attribute name="par" type="Map"  />
```

Comment:
good thing to notice. In the future please name your events something mnemonic that references the event's goal